### PR TITLE
Components

### PR DESCRIPTION
**Pull request by gjuliao**
## Requirements
- Delete boiler plate
- Create Components directory
- Create calculator design

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,10 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App container '>
+      <Calculator />
```
Comment (Qoosim) [inline src/App.js]: - [ ] It is required that all the components should be class-based components. Kindly change your App.js from functional to class-based component.

project-rqt

File: src/components/BtnNumbers.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+const BtnNumbers = ({ item }) => (
+  <div>
+    <button className='btn btn-secondary buttons' type='button'>
+      {item}
+    </button>
+  </div>
+);
+
+export default BtnNumbers;
```
Comment (Qoosim) [inline src/components/BtnNumbers.js]: - [ ] It is required that all the components should be class-based components. Kindly change this component from functional to class-based component.

project-rqt

File: src/components/BtnOperator.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+const BtnOperator = ({ item }) => (
+  <div>
+    <button className='btn btn-warning buttons' type='button'>
+      {item}
+    </button>
+  </div>
+);
+
+export default BtnOperator;
```
Comment (Qoosim) [inline src/components/BtnOperator.js]: - [ ] It is required that all the components should be class-based components. Kindly change this component from functional to class-based component.

project-rqt

File: src/components/Calculator.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable quotes, jsx-quotes, react/prop-types */
+
+import React from 'react';
+import BtnNumbers from './BtnNumbers';
+import BtnOperator from './BtnOperator';
+import Zero from './Zero';
+
+export default function calculator() {
+  return (
```
Comment (Qoosim) [inline src/components/Calculator.js]: - [ ] It is required that all the components should be class-based components. Kindly change your calculator component from functional to class-based component.

project-rqt

File: src/components/Zero.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+const Zero = ({ item }) => (
+  <div>
+    <button className='btn btn-secondary zero-btn' type='button'>
+      {item}
+    </button>
+  </div>
+);
+
+export default Zero;
```
Comment (Qoosim) [inline src/components/Zero.js]: - [ ] It is required that all the components should be class-based components. Kindly change this component from functional to class-based component.

project-rqt

File: src/App.js [hunk 6]
```diff
@@ -1,26 +1,10 @@
...
     </div>
   );
 }
```
Comment (Qoosim) [inline src/App.js]: - [ ] At this moment, your app does not display as expected on the web browser. Take a look at what I have on my browser. :point_down: 
your-app

- Expected display on the browser :point_down: 
expected

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @gjuliao, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 
-  Good README file but not too professional. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Kindly make the title in your README file to reflect the name of your project. Take a look :point_down: 

your-readme

- [ ] Also, remove other boilerplate files in your project e.g logo.svg etc.

File: src/components/BtnNumbers.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable quotes, jsx-quotes, react/prop-types */
```
Comment (ShadyShawkat) [inline src/components/BtnNumbers.js]: - [ ] Kindly make sure to update your readme, as requested by the previous reviewer.
- [ ] [OPTIONAL] Please refrain from using eslint disables as that defies the purpose of using linters in the first place.

File: src/components/Calculator.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+        </div>
+        <div className='d-flex'>
+          <Zero item='0' />
+          <BtnNumbers item='.' />
+          <BtnOperator item='=' />
+        </div>
+      </section>
+    );
+  }
+}
```
Comment (ShadyShawkat) [inline src/components/Calculator.js]: - [ ] As requested by the previous reviewer, please adjust the styling of the calculator to look more like the original design.
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @gjuliao, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Image

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @gjuliao 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**